docs/parameters.yaml

```diff
@@ -475,6 +475,12 @@ description: |+
         Capabilities: ["Reads", "PublicReads", "Writes", "Listings", "DirectReads"]
         SentinelLocation: demoproject_origin_A
     ```
+
+  There are additional configuration fields if Origin.StorageType == "s3".
+  - S3Bucket: [REQUIRED] See `Origin.S3Bucket` for details
```

Comment:
Ahh -- S3Bucket isn't actually required. When we want to export an entire endpoint like amazon's open data registry, we can't manually enumerate all of the buckets. In that case we get around it by omitting the bucket (which then becomes a part of the object name)